src/usr/local/buildpack/utils/environment.sh

```diff
@@ -72,24 +70,27 @@ function get_tool_version_env () {
 
 function setup_env_files () {
   # env helper, loads tool specific env
+  local install_dir
+  install_dir=$(get_install_dir)
+
   cat >> "$ENV_FILE" <<- EOM
-export BUILDPACK=1 USER_NAME="${USER_NAME}" USER_ID="${USER_ID}" USER_HOME="/home/${USER_NAME}"
+export BUILDPACK=1 USER_NAME="${USER_NAME}" USER_ID="${USER_ID}" USER_HOME="${USER_HOME}"
 
 # openshift override unknown user home
-if [ "\${EUID}" != 0 ]; then
+if [[ "\${EUID}" -ne 0 ]]; then
```

Comment:
So this now only works in bash? THis will thow syntax errors, when running default ubuntu dash